app-i18n/rime-data/rime-data-0.0.0.20230204.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+MY_PN="rime-essay"
+MY_COMMIT="e0519d0579722a0871efb68189272cba61a7350b"
+
+DESCRIPTION="Data resources for Rime Input Method Engine"
+HOMEPAGE="https://rime.im/ https://github.com/rime/plum"
+SRC_URI="https://github.com/rime/${MY_PN}/archive/${MY_COMMIT}.tar.gz -> ${MY_PN}-${PV}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="essay"
+KEYWORDS="~amd64 ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
```

Comment:
Maybe we should start with `~amd64 ~x86`

Reply:
These are all architecture independent packages (data only), so I decide to add more keywords to satisfy ibus-rime and possible fcitx-rime:4. I think it's fine.